microservices-connector/cmd/router/main.go

```diff
@@ -137,7 +149,12 @@ func getServiceURLByStepTarget(step *gmcv1alpha3.Step, svcNameSpace string) stri
 	return step.ServiceURL
 }
 
-func executeStep(step *gmcv1alpha3.Step, graph gmcv1alpha3.GMConnector, input []byte, headers http.Header) ([]byte, int, error) {
+func executeStep(
+	step *gmcv1alpha3.Step,
```

Comment:
Do we want to clean up this "gmc" in our next release to be "mc"?

Reply:
@mkbhanda sure, will do it.